webinterface/webinterface.py

```diff
@@ -18,7 +18,7 @@ def hello_world():
 @app.route('/register/<chat_id>')
 def register(chat_id):
     token = request.args.get('token')
-    return render_template('register_form.html', chat_id=chat_id, token=token)
+    return render_template('register_form.html', url_path=BotConf.url_path, chat_id=chat_id, token=token)
 
```

Comment:
Another idea:
Define url_path (and other global properties like the URL to libraries) as global variable using another function (insert above the other functions):
@app.before_request
def before_request():
    g.url_path = BotConf.url_path

The g object can be referenced directly in the templates: {{ g.url_path }}

You need to add g to the list of imports from flask in line 1